toolchain/lex/tokenized_buffer.h

```diff
@@ -395,18 +396,22 @@ class TokenizedBuffer : public Printable<TokenizedBuffer> {
         : kind_(kind),
           has_leading_space_(has_leading_space),
           token_payload_(payload),
-          byte_offset_(byte_offset) {
-      CARBON_DCHECK(payload >= 0 && payload < (2 << PayloadBits),
-                    "Payload won't fit into unsigned bit pack: {0}", payload);
-    }
+          byte_offset_(byte_offset) {}
 
     // A bitfield that encodes the token's kind, the leading space flag, and the
     // remaining bits in a payload. These are encoded together as a bitfield for
     // density and because these are the hottest fields of tokens for consumers
     // after lexing.
     TokenKind::RawEnumType kind_ : sizeof(TokenKind) * 8;
     bool has_leading_space_ : 1;
+    // Payload values are typically ID types for which we create at most one per
```

Comment:
```suggestion

    // Payload values are typically ID types for which we create at most one per
```

Whitespace suggestion due to the long comment.

Reply:
This makes the previous comment that's trying to talk about all three bit-fields less clear.

What do you think about moving this comment before all the bit-fields, with the other one?